sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_histogram_aggregates_parent_v1/query.sql

```diff
@@ -1,30 +1,18 @@
--- Query generated by: templates/clients_daily_histogram_aggregates.sql.py
-CREATE TEMPORARY FUNCTION get_keyval_pairs(y STRING)
-RETURNS ARRAY<STRING> DETERMINISTIC
-LANGUAGE js
-AS
-  '''
-          var z = new Array();
-          node = JSON.parse(y);
-          for (let [key, value] of Object.entries(node)) {
-            z.push(`${key}:${value}`);
-          }
-          return z;
-        ''';
-
+-- Query generated by: sql/telemetry_derived/clients_daily_histogram_aggregates_v1.sql.py
 CREATE TEMP FUNCTION udf_aggregate_json_sum(histograms ARRAY<STRING>) AS (
```

Comment:
I'm wondering if this UDF is worth having a mozfun UDF for? Something like `mozfun.hist.aggregate_values_sum`?

Reply:
Sounds reasonable, although I would expect the output to have keys of type INT64 rather than STRING, matching how the `values` array is returned by `mozfun.hist.extract`. If we promote this to mozfun, I'd like to make sure the signature is consistent with the rest of `mozfun.hist`, which would require a bit of refactoring in these queries.

Reply:
I'll do this in a follow-up PR